### ISSUE TITLE
junying/event_filter:  no delegations rewards

### ISSUE THREAD

**Issue by youngqqcn** [orientwalt/htdf#41]
no delegation rewards 

staking pool `not_bonded_tokens` is not changed
```
yqq@ubuntu:htdf$ hscli query staking pool
{
  "not_bonded_tokens": "5999990000000000",
  "bonded_tokens": "10000000000",
  "last_zero_block_height": "1",
  "amplitude_sine_function": "0",
  "cycle_sine_function": "1"
}

```

**Comment by youngqqcn** [comment]
https://github.com/orientwalt/htdf/blob/b7bd7deb7e95c93ca86e25b2e977cc5892201eab/x/distribution/keeper/store.go#L123-L129

b is bytes   [2, 16, 2]

but rewards type is `ValidatorHistoricalRewards`,  

https://github.com/orientwalt/htdf/blob/b7bd7deb7e95c93ca86e25b2e977cc5892201eab/x/distribution/types/validator.go#L21-L24

after `MustUnmarshalBinaryLengthPrefixed`   `rewards.CumulativeRewardRatio ` is nil

**Comment by bluesky0125** [comment]
this issue resolved since commit: 2877b6f5e1c76763df7870c9a16f1dc0df49f9c3
